src/sources/http_server.rs

```diff
@@ -187,7 +187,7 @@ impl SimpleHttpConfig {
         schema_definition
     }
 
-    fn get_decoding_config(&self) -> crate::Result<DecodingConfig> {
+    pub fn get_decoding_config(&self) -> crate::Result<DecodingConfig> {
```

Comment:
Not finding the reason this was changed to pub on the diff as-is (same for the other one changed in this PR. is that implicitly needed due to changes in the validation module ?

Reply:
Yep, I just dropped some of these. This code change went through a few different iterations.